modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -104,11 +104,15 @@ Runtime Fabric on VMs / Bare Metal requires one of the following operating syste
 * Red Hat (RHEL) v7.4, v7.5, v7.6, v7.7, v7.8, v7.9, v8.0, v8.1, v8.2
 * CentOS v7.4, v7.5, v7.6, v7.7, v7.8, v7.9, v8.0, v8.1, v8.2
 
+Use the same operating system for each node. If you attempt to install Runtime Fabric on VMs / Bare Metal on 
+a different operating system version or distribution, the installer fails.
+
 [IMPORTANT]
-Currently, there is a known Control Groups (cgroups) memory leaking issue, which can lead to pod creation failure or node malfunction. Ensure that you are using Linux kernel version 3.10.0-1075 or later, and refer to https://help.mulesoft.com/s/article/RTF-How-to-Resolve-the-Cgroup-Memory-Leaking-Issue-in-Runtime-Fabric[RTF - How to Resolve the Cgroup Memory Leaking Issue in Runtime Fabric] for additional information.
+====
+There is a known control groups (cgroups) memory leak issue which can lead to pod creation failure or node malfunction in RHEL 7.x. See  https://help.mulesoft.com/s/article/RTF-How-to-Resolve-the-Cgroup-Memory-Leaking-Issue-in-Runtime-Fabric[RTF - How to Resolve the Cgroup Memory Leaking Issue in Runtime Fabric] for additional information.
```

Comment:
Add CentOS at end of first sentence:

"..creation failure or node malfunction in RHEL **and CentOS 7**."

Reply:
Suggest naming the link something like "this KB article" instead of the full name of the article.

Reply:
Fixed. Not sure about the style for the KB links. I'll follow up on that, but for now I incorporated your suggestion.